### DISCUSSION TITLE
Overlay text transparent at a specific position

### DISCUSSION THREAD

**Discussion by tbmuc** [ImageMagick/ImageMagick#2276] [Help]
Hello,
I would like to place an overplayed text on this image and played a little around but did not find a solution.

The text should be centered around the position 5294, 1195 . The fontsize should be 80. 

How to do that?

![IH32tp1000l](https://user-images.githubusercontent.com/48103248/87650199-90923200-c751-11ea-88bf-eb1037aa8035.jpg)

**Comment by snibgo** [comment]
What version of IM, on what platform? I will assume v7 on Windows.

Your sample image is not large enough for a text at that position, I will assume you want the bottom-left of the text to be at (529,119).
```
magick in.jpg -pointsize 80 -annotate +529+119 "Hello world" out.jpg
```
See https://imagemagick.org/script/command-line-options.php#annotate

**Reply by tbmuc** [reply]
Thank you very much for your helpful answer. I'm using IM 7 on Windows and the origional image size is 8834x5534 pixel.

The current statement
magick tmp1.jpg -density 300 -pointsize 80 -gravity sw -annotate +5294+1195 "Hello world" out.jpg

How do I get the center of the text, both horizontally and vertically to this position? Text length ist not fixed but its restricted to one line.
